src/components/link/Link.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image'
+
+type LinkProps = {
+  url?: string
+  name: string
+  icon: string
```

Comment:
Why `icon` is a mandatory property?
I'll expect to display a simple `link` without any icon.